### PR TITLE
Fix/midrc

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
Link to Jira ticket if there is one:

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# staging.midrc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [file_name] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# staging.midrc.org
 :x: etlMapping.yaml - gitops.json mismatch
- Field error: Field [file_name] in explorerConfig.table.fields not found in etlMapping

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services are being downgraded
- indexd
- revproxy
- peregrine
- sheepdog
- pidgin
- fence
## Deployment changes
- fence
  - Configure the `ALLOWED_DATA_UPLOAD_BUCKETS` setting to allow users to upload to buckets other than `DATA_UPLOAD_BUCKET` (uc-cdis/fence#1051)
  - B/c the default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP, you MUST ensure that an instance that should comply with FedRAMP does NOT override this fence configuration (e.g. it should be 15 minutes) `SESSION_TIMEOUT: 900` (uc-cdis/fence#1040)
  - Requires a Fence DB migration (`Client.redirect_uri` is now optional) (uc-cdis/fence#1033)
## Breaking changes
- fence
  - Default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP (uc-cdis/fence#1040)
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (uc-cdis/indexd#336)
- portal
  - Replaced loading  the data in  `iframes` in Workspace page with Redirect component. (uc-cdis/data-portal#1067)
  - Deprecating the use of `default-workspaces` i. e an individual commons will no longer be able to deploy standalone jupyterhub images, they may use [hatchery service](https://github.com/uc-cdis/hatchery) instead. (uc-cdis/data-portal#1067)
  - `studyRegistrationConfig` in `gitops.json` must now be placed within the `regsitrationConfigs.features` block [[Refer this PR](https://github.com/uc-cdis/cdis-manifest/pull/5095)] (uc-cdis/data-portal#1067)

File: staging.midrc.org/etlMapping.yaml [hunk 1]
```diff
@@ -96,9 +38,6 @@ mappings:
           - name: object_id
             src: object_id
             fn: set
-          - name: file_name
```
[the remark this answers is not in the document]
Comment (thanh-nguyen-dang) [reply staging.midrc.org/etlMapping.yaml:99]: @cgmeyer  said that we don't need that anymore

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services are being downgraded
- fence
- pidgin
- peregrine
- sheepdog
- indexd
- revproxy
## Deployment changes
- fence
  - Configure the `ALLOWED_DATA_UPLOAD_BUCKETS` setting to allow users to upload to buckets other than `DATA_UPLOAD_BUCKET` (uc-cdis/fence#1051)
  - B/c the default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP, you MUST ensure that an instance that should comply with FedRAMP does NOT override this fence configuration (e.g. it should be 15 minutes) `SESSION_TIMEOUT: 900` (uc-cdis/fence#1040)
  - Requires a Fence DB migration (`Client.redirect_uri` is now optional) (uc-cdis/fence#1033)
## Breaking changes
- fence
  - Default session expiration changed from 30 minutes to 15 minutes to comply with FedRAMP (uc-cdis/fence#1040)
- indexd
  - Ensure you are using Fence>=2021.10. Remove deprecated GA4GH access endpoints. these have existed (and been used) from the Fence microservice since last year. See cloud automation change [here](https://github.com/uc-cdis/cloud-automation/blob/f10f04f7957eb2ba179504f7df9d6429f7298835/gen3/bin/kube-setup-revproxy.sh#L78-L79) (uc-cdis/indexd#336)
- portal
  - Replaced loading  the data in  `iframes` in Workspace page with Redirect component. (uc-cdis/data-portal#1067)
  - Deprecating the use of `default-workspaces` i. e an individual commons will no longer be able to deploy standalone jupyterhub images, they may use [hatchery service](https://github.com/uc-cdis/hatchery) instead. (uc-cdis/data-portal#1067)
  - `studyRegistrationConfig` in `gitops.json` must now be placed within the `regsitrationConfigs.features` block [[Refer this PR](https://github.com/uc-cdis/cdis-manifest/pull/5095)] (uc-cdis/data-portal#1067)

**Pull request merged**